client/extensions/woocommerce/components/reading-widget/index.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import emailValidator from 'email-validator';
+import { localize } from 'i18n-calypso';
+import React, { Component, PropTypes } from 'react';
+import ExternalLink from 'components/external-link';
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc